Scrum 61 make campfire fire pixelated

Panda1941 opened the pull request:
slight adjustments to the safe area campfire particles.

Acceptance criteria mostly met, except for particles being pixel alligned, I couldnt figure it out. Performance doesn't seem to be impacted.

Any complants, just write in the comments bellow.

Augis154 commented:
Maybe minimize smoke a little bit but idk

Augis154 commented:
> > Maybe minimize smoke a little bit but idk
> 
> by minimize you mean make the particles smaller or make it so there are fewer of them?

Fewer of them.

Panda1941 commented:
Reduced the number of smoke particles
Added some higher flying fire particles (looks a bit fancier imo xdd)
Adjusted the light values for both the player and the campfire

Panda1941 commented:
if everything is fine, go ahead and merge it, otherwise let me know
